installer/ansible/roles/dashboard-installer/scenarios/container.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: install docker-py package with pip when enabling containerized deployment
+  pip:
+    name: docker-py
+
+- name: run dashboard containerized service
+  docker_container:
+    name: dashboard
+    image: "{{ dashboard_docker_image }}"
+    state: started
+    network_mode: host
+    restart_policy: always
+    env:
+      OPENSDS_AUTH_URL: "http://{{ host_ip }}/identity"
+      OPENSDS_HOTPOT_URL: "http://{{ host_ip }}:50040"
+      OPENSDS_GELATO_URL: "http://{{ host_ip }}:8089"
+      SODA_DELFIN_URL: "{{ soda_delfin_url }}"
+      OPENSDS_ORCHESTRATION_URL: "http://{{ host_ip }}:5000"
+      OPENSDS_S3_URL: "http://{{ host_ip }}:8090"
```

Comment:
All the env are needed for Delfin?!
Why should it be called OpenSDS or SODA ?
Any env variable at SODA level should be part of SODA installer / dashboard not delfin.
Delfin should handle delfin env vars

Reply:
Done